codebuild/bin/install_awslc.sh

```diff
@@ -35,12 +35,9 @@ fi
 
 # These tags represents the latest versions that S2N is compatible
 # with. It prevents our build system from breaking when AWS-LC
-# is updated, last done on 2023-02-22.
-if [ "$IS_FIPS" == "1" ]; then
-  AWSLC_VERSION=AWS-LC-FIPS-1.0.3
-else
-  AWSLC_VERSION=v1.36.0
-fi
+# is updated.
+AWSLC_VERSION=v1.47.0
```

Comment:
nit - this is a bit unrelated to this PR since this was already pinned, but if AWS-LC has a release that breaks our build, wouldn't we rather know about that rather than obscure it by pinning the version?

Though, we effectively pin the version anyway since the docker image isn't updated very often. But assuming that becomes more frequent, would it be reasonable to just always use the latest AWS-LC release?

Reply:
I'm game, yes, totally reasonable. The PR is about not pinning, so arguably related.